### DISCUSSION TITLE
Private by Default

### DISCUSSION THREAD

**Discussion by stephen-hill** [xenocrat/chyrp-lite#277] [Q&A] [answered]
Hello

Is it possible to configure chyrp-lite so that, when creating a new text, the "Status" drop down defaults to "Private" rather than the current default "Public".

Many Thanks
Stephen

**Comment by xenocrat** [comment, marked as answer]
Not right now, but that's a good feature. Will be added for the next release!

**Comment by stephen-hill** [comment]
@xenocrat  Thank you :)
